### PR TITLE
docs: HTTP API Status Codes

### PR DESCRIPTION
**Pull request by lidel**
This PR updates HTTP API docs with a newly generated version from https://github.com/ipfs/http-api-docs/pull/23, which adds docs about status codes in HTTP API

Any issues should be reported in the above issue.
This should be merged if upstream PR gets accepted.

### CODE REVIEW

**Comment by lidel** [comment]
@hsanjuan any idea why cluster panics in https://circleci.com/gh/ipfs/docs/621 ?

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9c4c80]

goroutine 84 [running]:
main.main.func6.1(0xc00002b574, 0xc0004d6280, 0xc0000b4780)
	github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-ctl/main.go:436 +0x320
created by main.main.func6
	github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-ctl/main.go:417 +0x60f
Exited with code 2
```

**Comment by hsanjuan** [comment]
> @hsanjuan any idea why cluster panics in https://circleci.com/gh/ipfs/docs/621 ?
> 
> ```
> panic: runtime error: invalid memory address or nil pointer dereference
> [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9c4c80]
> 
> goroutine 84 [running]:
> main.main.func6.1(0xc00002b574, 0xc0004d6280, 0xc0000b4780)
> 	github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-ctl/main.go:436 +0x320
> created by main.main.func6
> 	github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-ctl/main.go:417 +0x60f
> Exited with code 2
> ```

I don't know, but this seems to have been fixed...

File: content/reference/api/http.md [hunk 1]
```diff
@@ -20,17 +20,21 @@ your program — it allows you to maintain peer connections that are longer
...
-the [ipfs-http-api-docs](https://github.com/ipfs/ipfs-http-api-docs)
-repository on GitHub.
+<aside class="alert alert-info">
+  This document was autogenerated from go-ipfs.<br/>
+  For issues and support, check out
+  the <a href="https://github.com/ipfs/ipfs-http-api-docs" target="_blank">ipfs-http-api-docs&nbsp;<i class="fas fa-external-link-square-alt fa-sm"></i></a> repository on GitHub.
+</aside>
 
-## Getting started
+## Getting Started
```
Comment (jessicaschilling) [inline content/reference/api/http.md]:
```suggestion
## Getting started
```
In line with our current use of caps

**Comment by jessicaschilling** [comment]
@lidel @hsanjuan I'm still seeing the same CircleCI failure. Any ideas?

**Comment by hsanjuan** [comment]
@olizilla will update the deploy docker image thing. Then we should just re-trigger the build and it should not panic anymore.

**Comment by jessicaschilling** [comment]
@olizilla -- can you please let me know when you've deployed the Docker image @hsanjuan refers to in this issue, so this can be merged? Thank you!

**Comment by olizilla** [comment]
It's all good to go. We couldn't connect to cluster. It's fixed now.

**Comment by jessicaschilling** [comment]
@olizilla Thank you so much!

**Pull request merged**